app/controllers/application_controller.rb

```diff
@@ -262,7 +270,9 @@ def check_student_consent
   # a hidden param :after_sign_in_path to the sign in form.
   def after_sign_in_path_for(resource)
     redirect_path = root_path
-    if current_user.portal_student
+    if ENV['RESEARCHER_REPORT_ONLY']
+      redirect_path = learner_report_path
```

Comment:
So, this is added so we don't need to use `after_sign_in_path`, right?
Would `after_sign_in_path` cause some problems? Again, mostly asking so I understand it better.